watsong/spotify.py

```diff
@@ -88,7 +87,7 @@ def cache(album_descriptions: List[AlbumDescription]) -> None:
     set_memo(album_tracks_memo, "tracks")
 
     songs, err = get_songs(album_descriptions)
-    for songs_chunk in util.chunks(iter(songs), 100):
```

Comment:
Why not 100?

Reply:
At the time I was thinking that 100 is too big and we would miss a higher % of the time but actually that's a dumb way of thinking cause we will just almost miss less frequently with 100, I'll change it back